dataplane/windows/endpoint_mgr.go

```diff
@@ -267,6 +268,27 @@ func (m *endpointManager) ProcessIpSetUpdate(ipSetId string) {
 	}
 }
 
+// ProcessIpSetUpdate is called when a IPSet has changed. The ipSetsManager will have already updated
+// the IPSet itself, but the endpointManager is responsible for requesting all impacted policy sets
+// to be updated and for marking all impacted endpoints as pending so that updated policies can be
+// pushed to them.
+func (m *endpointManager) ProcessIpSetUpdate(ipSetId string) {
+	log.WithField("ipSetId", ipSetId).Debug("Requesting PolicySetsDataplane to process the IP set update")
+	updatedPolicies := m.policysetsDataplane.ProcessIpSetUpdate(ipSetId)
+	m.refreshPendingWlEpUpdates(updatedPolicies)
+}
+
+// ProcessPolicyProfileUpdate is called when a Policy or Profile has changed. The policySetsDataplane will have
+// already updated the Policy or Profile itself, but the endpointManager is responsible for requesting all impacted
```

Comment:
Nit: I think we're only marking the endpoints for update, not requesting impacted policy set updates here.